lib/mix/tasks/appsignal.install.ex

```diff
@@ -28,6 +28,7 @@ defmodule Mix.Tasks.Appsignal.Install do
         activate_config_for_env("dev")
         activate_config_for_env("stag")
         activate_config_for_env("prod")
+        deactivate_config_for_env("test")
```

Comment:
Maybe we can check beforehand if any env specific config files exist and if so set `active: false` in `appsignal.exs` so we don't have explicitly to deactivate the test env? or would that become to complicated?